CWE/technologies/commerce_tech.txt

```diff
@@ -1192,7 +1192,7 @@ late_fordism = {
 	year = 1946
 	cost = 3600
 	
-	activate_building = steel_factory 
+	#activate_building = steel_factory 
```

Comment:
This cannot be commented out because steel factory is not default enabled and is activated by this tech.

Reply:
Doesn't steel_factory_manufacturing unlock the steel factory if electricity tech is researched?

Reply:
Sorry my bad, I totally forgot about that invention.